packaging/linux/amazon-cloudwatch-agent.spec

```diff
@@ -100,19 +86,6 @@ if ! id cwagent >/dev/null 2>&1; then
     echo "create user cwagent, result: $?"
 fi
 
-if ! grep "^aoc:" /etc/group >/dev/null 2>&1; then
```

Comment:
What we should do here is remove the aoc user if it exists since we don't need it anymore. We shouldn't just leave the user there on the host

Reply:
Thanks for the suggestion! I will make changes and also validate changes work as expected